lib/taglessFinal/dune

```diff
@@ -20,18 +20,16 @@
       11.0
       (progn
        (run ./configure)
-       (run make libpython3.11.a))))
+       (run make))))
     (copy vendor/libpython3.11.a libpython.a)
     (copy vendor/pyconfig.h pyconfig.h)))))
 
 (library
  (name taglessFinal)
  (package pyre-ast)
  (libraries base)
- (no_dynlink)
  (foreign_stubs
   (language c)
   (names binding)
   (flags :standard -Ivendor/Include -I.))
- (c_library_flags -lpython -lutil -lpthread)
- (foreign_archives python))
+ (c_library_flags -lutil -lpthread -lpython -L%{project_root}/_build/default/lib/taglessFinal/))
```

Comment:
This is hacky! I should be able to get the outputs of the rule in the context of the C build without adding the `-L` command.

Reply:
Yeah this creates hard dependency on where the build directory is. Any chance we could get the output without hardcoding the path?

Reply:
That's what I'm looking for but don't know of a way at the moment 😢. Hopefully there's a way and someone in https://github.com/ocaml/dune/issues/8335 can show me.